tools/amd_build/build_amd.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import sys
+import argparse
+
+sys.path.append(os.path.realpath(os.path.join(
+    os.path.dirname(__file__),
+    os.path.pardir,
+    os.path.pardir,
+    'third_party')))
+
+from hipify import hipify_python
+
+parser = argparse.ArgumentParser(description='Top-level script for HIPifying, filling in most common parameters')
```

Comment:
It is generally more idiomatic to put all the logic of a runnable Python entry-point script in a `def main():` function and then invoke it with `if __name__ == "__main__": main()`.

Reply:
Updated the code as suggest and committed to hipify [repo](https://github.com/ROCmSoftwarePlatform/hipify-torch)